src/wallet/wallet.ts

```diff
@@ -1099,6 +1102,65 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     return parseInt(parts[5], 10);
   }
 
+
+  /**
+   * Get mint authorities
+   * Uses the getTxOutputs API method to return one or many mint authorities
+   *
+   * @param tokenId of the token to select the authority utxo
+   * @param options Object with custom options.
+   *  {
+   *    'many': if should return many utxos or just one (default false),
+   *    'skipSpent': if should not include spent utxos (default true)
+   *  }
+   *
+   * @return Array of objects with {txId, index, address} of the authority output. Returns an empty array in case there are no tx outputs for this type
+   **/
+  async getMintAuthority(tokenId: string, options: { many?: boolean, skipSpent?: boolean } = {}): Promise<AuthorityTxOutput[]> {
+    const newOptions = Object.assign({ many: false, skipSpent: true }, options);
+    const { txOutputs } = await walletApi.getTxOutputs(this, {
+      tokenId,
+      authority: TOKEN_MINT_MASK,
+      skipSpent: newOptions.skipSpent,
+      maxOutputs: newOptions.many ? undefined : 1,
+    });
+
+    return txOutputs.map((txOutput) => ({
+      txId: txOutput.txId,
+      index: txOutput.index,
+      address: txOutput.address,
+    }));
+  }
+
+  /**
+   * Get melt authorities
+   * Uses the getTxOutputs API method to return one or many mint authorities
```

Comment:
melt*

Reply:
Fixed in b058f570f3e8fc3012f6ac6d3f112d8ff4cf5410